test/clj/quil/snippet.clj

```diff
@@ -81,3 +115,28 @@
                       {}))))
   (t/is true))
 
+(t/deftest ^:manual
+  all-cljs-snippets-produce-expected-output
+  (let [driver (e/firefox)
+        threshold 0.0001]
+    (e/go driver (str "http://localhost:3000/test.html"))
+    (let [elements (e/query-all driver {:tag :option})
+          names    (doall (map #(e/get-element-text-el driver %) elements))
+          c        (count elements)]
```

Comment:
I don't think you need to alias `(count elements)` as c given that you use `c` only once. Maybe just do `(dotimes [i (count elements)] ...)`?

Reply:
done